N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
Note that N3 is really an implementation of Guha's thesis on contexts [Contexts 􏰄 A Formalization and Some Applications](http://www-formal.stanford.edu/buvac/guha-thesis.ps)